echo_handler.cpp

```diff
@@ -0,0 +1,26 @@
+// Based Off http://www.boost.org/doc/libs/1_55_0/doc/html/boost_asio/example/cpp11/http/server/request_handler.cpp
```

Comment:
Nice job including the source of your file! Do we know if there is anything else we need to cite when including these sources?